docs/_partners/integration-quality.md

```diff
@@ -123,3 +131,26 @@ Approaching active user retention with a long-term strategy can help maintain a
 * Update the integration regularly with features as your platform evolves. [Invite stakeholders to your integration](https://platform.zapier.com/quickstart/invite-team-member) to give them admin or read-only access to insights, metrics, and feedback to prioritize and align improvements. 
+
+
+## Integration Insights Definitions
+
+The following includes definitions for each of the metrics provided in the integration Dashboard:
+
+| **Metric**                  | Definition                                                                              | Available Filters   |
+|-----------------------------|-----------------------------------------------------------------------------------------|---------------------|
+| **New users**               | Total number of *new users using the integration over a selected period of time.<br><br> *A new user = first time a user creates and activates a Zap with this integration. | <ul><li>Last 7 days</li><li>Last 30 days</li><li>Last 90 days</li></ul> |
+| **Bugs & feature requests** | Total number of open bugs and feature requests for the integrations.                    |                     |
+| **Daily active users**      | Number of users who had a Zap *activated in a day.<br><br> *Activated = Zap is on and successfully ran at least one task. | <ul><li>Last 7 days</li><li>Last 30 days</li><li>Last 90 days</li></ul> |
+| **Monthly active users**    | Total number of users who had a Zap *activated in a given month.<br><br> *Activated = Zap is on and successfully ran at least one task.                         | By year                  |
+| **Active Zaps**             | Total number of *active Zaps over a selected period of time.<br><br> *Active Zaps = number of Zaps currently on and using the integration.                  | <ul><li>Last 7 days</li><li>Last 30 days</li><li>Last 90 days</li></ul>  |
+| **Daily active Zaps**       | Number of *active Zaps in a day.<br><br> *Active Zaps = number of Zaps currently on and using the integration. | <ul><li>Last 7 days</li><li>Last 30 days</li><li>Last 90 days</li></ul>                   |
+| **Active users retention**  | The percentage of active users *retained each month.<br><br> *Retention = users who have at least one successful task executed in a given month.                  |                              |
+| **Usage by Trigger**        | Number of current live Zaps (Zaps turned on), paused Zaps (Zaps turned off), and total Zaps for each trigger.  | By integration version                       |
+| **Usage by Action**         | Number of current live Zaps (Zaps turned on), paused Zaps (Zaps turned off), and total Zaps for each action.   | By integration version                       |
```

Comment:
Considering that this table is huge and includes nested HTML, it might be more readable to just use HTML table syntax directly